mox/src/lib.rs

```diff
@@ -12,11 +12,11 @@ use syn::{
 };
 use syn_rsx::{NodeName, NodeType};
 
-/// Accepts an XML-like expression and expands it to builder method calls.
```

Comment:
I think this is still accurate, maybe it should say "builder-like method calls"?

Reply:
Fixed